roxie/ccd/ccdcontext.cpp

```diff
@@ -2581,33 +2571,26 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
 
     void doPostProcess()
     {
-        if (workUnit || statsWu)
+/*        if (workUnit || statsWu)
```

Comment:
remove the old code, and leave a comment?

Reply:
removed. Not sure that a comment would be especially helpful